Readme.md

```diff
@@ -1,33 +1,47 @@
 # World Explorer
 
-GM tool for hexcrawl campaigns that allows displaying a second manual fog of war (with color or image) only over the background layer, while keeping the grid, tokens, and tiles visible. Grid tiles can be removed manually by the GM to reveal the underlying map on successing scouting or mapping checks. Enable in scene configuration.
+GM tool for hexcrawl campaigns that allows displaying a second manual fog of war (with color or image) only over the background layer while keeping the grid visible. You can choose what else this layer covers or not: drawings, (overhead) tiles, and tokens. Grid spaces can be revealed or made partially visible. The GM can always do this manually, but you can also enable auto-revealing areas explored by friendly tokens.
```

Comment:
Its ok to remove tiles from the first sentence, since tiles being visible is optional based on placement, but tokens needs to be mentioned. Most other fog of war modules actually obscure the tokens.